### PR TITLE
Issue 2018/02 (obsolete)

### PR DESCRIPTION
**Pull request by badmutex**
Target date: 2018-02-01

Help needed: please comment below with suggestions

### CODE REVIEW

**Comment by AndersonTorres** [comment]
A thing we need to track: Google Summer of Code!

File: content/2018/02-life-before-death.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#02 - Life before death, strength before weakness, ...
```
Comment (garbas) [inline content/2018/02-life-before-death.rst:1]: ❤️ the quote :)

**Comment by garbas** [comment]
I merged this manually to master and open a new PR from `master` to `production` -> https://github.com/NixOS/nixos-weekly/pull/46

**Comment by garbas** [comment]
@AndersonTorres how do you propose to keep an eye on this? Is Nix participating this year? Which link(s) we should be checking out?

**Comment by AndersonTorres** [comment]
@garbas , this is [the official Github link](https://github.com/nix-community/google-summer-of-code). @zimbatm is now discussing about it, NixOS participating on GSOC, in the mailing list.

**Comment by badmutex** [comment]
@AndersonTorres thanks for the update. I added GSoC to #46 and you can see the preview [here](http://weekly.nixos.org/preview/2018/02-life-before-death.html) if you like.

**Pull request closed without merging**